In-house patch to remove the dependency on argparse.  Since we use
Python 2.7 only for OpenStack, argparse is already included.  If this
line is present, stevedore will fail to find the pip package for it,
causing a cascade of failures to other tools.

Drop the required version of keystoneclient to be the same as Kilo
global reqs.  This version of cinderclient was cherry picked to
pull in features that were missed because the cinder team did not
cut a cinderclient to match the kilo release.

--- python-cinderclient-1.3.1/requirements.txt.orig	2015-11-19 14:55:25.974425490 -0700
+++ python-cinderclient-1.3.1/requirements.txt	2015-11-19 14:56:46.549580080 -0700
@@ -2,9 +2,8 @@
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
 pbr<2.0,>=0.11
-argparse
 PrettyTable<0.8,>=0.7
-python-keystoneclient>=1.6.0
+python-keystoneclient<1.4.0,>=1.2.0
 requests>=2.5.2
 simplejson>=2.2.0
 Babel>=1.3

--- python-cinderclient-1.3.1/python_cinderclient.egg-info/requires.txt.orig	2015-11-19 14:57:35.202730172 -0700
+++ python-cinderclient-1.3.1/python_cinderclient.egg-info/requires.txt	2015-11-19 14:57:18.307302488 -0700
@@ -1,7 +1,6 @@
 pbr<2.0,>=0.11
-argparse
 PrettyTable<0.8,>=0.7
-python-keystoneclient>=1.6.0
+python-keystoneclient<1.4.0,>=1.2.0
 requests>=2.5.2
 simplejson>=2.2.0
 Babel>=1.3
